diff -urN squidclamav.orig/paths.h squidclamav/paths.h
--- squidclamav.orig/paths.h	2005-12-14 22:07:02.000000000 +0100
+++ squidclamav/paths.h	2005-12-31 09:51:35.000000000 +0100
@@ -25,10 +25,10 @@
 
 /*************  Log File Locations (must be writable by **********/
 /*************  the user given in the squid.conf file   **********/
-#define LOG_FILE "/usr/local/squidclamav/logs/squidclamav.log"
+#define LOG_FILE "/var/log/squid/squidclamav.log"
 
 /*************  Configuration file locations  ***********/
-#define CONFIG_FILE "/usr/local/squidclamav/etc/squidclamav.conf"
+#define CONFIG_FILE "/etc/squid/squidclamav.conf"
 
 /************* Proxy to use *************/
 #define MY_PROXY "http://127.0.0.1:3128"
