diff -ur squidclamav-6.2.orig//configure.in squidclamav-6.2/configure.in
--- squidclamav-6.2.orig//configure.in	2011-02-27 11:04:28.000000000 +0100
+++ squidclamav-6.2/configure.in	2011-03-21 11:13:33.285487022 +0100
@@ -49,7 +49,7 @@
 )
 
 if test -z "$cicapdir"; then
-    if c-icap-config >& /dev/null; then
+    if c-icap-config > /dev/null; then
         cicapflags=`c-icap-config --cflags`
         cicaplibs=`c-icap-config --libs`
         cicapetc=`c-icap-config --configdir`
