diff -ur squidclamav-6.2.orig//etc/Makefile.am squidclamav-6.2.mod//etc/Makefile.am
--- squidclamav-6.2.orig//etc/Makefile.am	2010-02-19 17:40:28.000000000 +0100
+++ squidclamav-6.2.mod//etc/Makefile.am	2011-03-21 12:44:04.244275635 +0100
@@ -1,5 +1,5 @@
 EXTRA_DIST = squidclamav.conf
-CFGINST = @CFGDIR@
+CFGINST = @CONFDIR@
 
 install-data-local:
 	$(mkinstalldirs) $(DESTDIR)$(CFGINST)
diff -ur squidclamav-6.2.orig//etc/squidclamav.conf squidclamav-6.2.mod//etc/squidclamav.conf
--- squidclamav-6.2.orig//etc/squidclamav.conf	2011-02-27 11:32:12.000000000 +0100
+++ squidclamav-6.2.mod//etc/squidclamav.conf	2011-03-21 12:45:48.662389568 +0100
@@ -17,13 +17,13 @@
 redirect http://proxy.domain.dom/cgi-bin/clwarn.cgi
 
 # Path to the squiGuard binary if you want URL filtering
-#squidguard /usr/local/squidGuard/bin/squidGuard
+#squidguard /usr/bin/squidGuard
 
 # Path to the clamd socket, use clamd_local if you use Unix socket or if clamd
 # is listening on an Inet socket, comment clamd_local and set the clamd_ip and
 # clamd_port to the corresponding value.
-clamd_local /tmp/clamd
-#clamd_ip 192.168.1.5,127.0.0.1
+clamd_local /var/lib/clamav/clamd.socket
+#clamd_ip 127.0.0.1
 #clamd_port 3310
 
 # Set the timeout for clamd connection. Default is 1 second, this is a good
diff -ur squidclamav-6.2.orig//src/squidclamav.h squidclamav-6.2.mod//src/squidclamav.h
--- squidclamav-6.2.orig//src/squidclamav.h	2010-10-21 23:23:47.000000000 +0200
+++ squidclamav-6.2.mod//src/squidclamav.h	2011-03-21 12:44:15.448121918 +0100
@@ -42,7 +42,7 @@
 
 /* squidclamav.h */
 /*************  Default configuration file location  ***********/
-#define CONFIG_FILE "/etc/squidclamav.conf"
+#define CONFIG_FILE "/etc/c-icap/squidclamav.conf"
 
 /************* Proxy configuration *************/
 #define PROXY_SERVER "127.0.0.1"
